tests/CMakeLists.txt

```diff
@@ -71,9 +71,13 @@ else() # !PROJECT_IS_TOP_LEVEL
   endif()
   if(LIB3MF_FOUND)
     set(ENABLE_LIB3MF_TESTS TRUE)
+  else()
+    set(ENABLE_LIB3MF_TESTS FALSE)
   endif()
```

Comment:
Not quite sure I understand the motivation for doing this. In traditional programming languages, it's tempting to force variables to have a certain type with well-defined values. CMake is kind of built around a certain definition of truthy/falsey, and it feels a bit like you're trying to engineer yourself around CMake instead of embracing it :)

Reply:
If we can remove that file, I'm all for it. I'm not familiar with any corner cases handled by that script though..

Yeah, running nektos/act on a number of variants could be interesting. Challenge is that there are hundreds of combinations of variants : /